packages/esm-patient-chart-app/src/patient-chart/chart-review/chart-review.component.tsx

```diff
@@ -46,8 +46,11 @@ const ChartReview: React.FC<ChartReviewProps> = ({ patientUuid, patient, view })
     )
     .flat();
   const dashboards = ungroupedDashboards.concat(groupedDashboards) as Array<DashboardConfig>;
+  const filteredDashboards = dashboards.filter((dashboard) => {
+    if ('path' in dashboard) return dashboard;
+  });
```

Comment:
```suggestion
  const filteredDashboards = dashboards.filter(dashboard => 'path' in dashboard);

```

Question: @pirupius Shouldn't our filter only give a pass for dashboards with a valid path? Like a truthy one?

```js
 const filteredDashboards = dashboards.filter(dashboard => dashboard.path);
```

I think we should also log to console all excluded dashboards and flag them as missing a `path` property

Reply:
The dashboards without paths are mostly flags and other components that necessarily don't appear in the sidenav so there's no need to add a console log